src/__tests__/views/article.spec.js

```diff
@@ -0,0 +1,57 @@
+import React from 'react';
+import { mount } from 'enzyme';
+import { Provider } from 'react-redux';
+import configureStore from 'redux-mock-store';
+import thunk from 'redux-thunk';
+import Article from '../../views/Article/Article';
+
+describe('Article', () => {
+  const mockStore = configureStore([thunk]);
+  let component;
+  let article = {
+    title: 'test',
+    description: 'description',
+    images: ['image'],
+    body: 'body',
+  };
+
+  it('renders correctly', () => {
```

Comment:
@Dubby20 this description can be better put. You can describe what you're testing

Reply:
I've edited it to be more descriptive. Thanks